ios/RainbowText.m

```diff
@@ -92,7 +92,7 @@ - (void) animate {
 
   NSString *newValue;
   if (_isSymbolStablecoin) {
-    newValue = [NSString stringWithFormat:@"$%@", [_fmt stringFromNumber:@(value)]];
+    newValue = [NSString stringWithFormat:@"%@ %@", [_fmt stringFromNumber:@(value)], _symbol];
```

Comment:
in this case, given that we are no longer doing anything special for stablecoins, can remove the if else